tensorflow_datasets/scripts/templates/dataset.mako.md

```diff
@@ -1,283 +1,310 @@
-
-def get_config_description(builder):
-  return builder.builder_config.description
-
-def get_homepage(builder):
-  return builder.info.homepage
-
-def get_source(builder):
-  return True  # Always common to all configs
-
-def get_versions(builder):
-  return tuple((str(v), v.description) for v in builder.versions)
-
-def get_download_size(builder):
-  return builder.info.download_size
-
-def get_dataset_size(builder):
-  return builder.info.dataset_size
-
-def get_manual(builder):
-  return builder.MANUAL_DOWNLOAD_INSTRUCTIONS
-
-def get_autocache(builder):
-  return build_autocached_info(builder)
-
-def get_splits(builder):
-  return tuple(
-      (str(s.name), int(s.num_examples)) for s in builder.info.splits.values()
-  )
-
-def get_features(builder):
-  return repr(builder.info.features)
-
-def get_supervised(builder):
-  return builder.info.supervised_keys
-
-def get_citation(builder):
-  return builder.info.citation
-
-all_sections = [
-    Section(get_description, display_description),
-    Section(get_config_description, display_config_description),
-    Section(get_homepage, display_homepage),
-    Section(get_source, display_source),
-    Section(get_versions, display_versions),
-    Section(get_download_size, display_download_size),
-    Section(get_dataset_size, display_dataset_size),
-    Section(get_manual, display_manual),
-    Section(get_autocache, display_autocache),
-    Section(get_splits, display_splits),
-    Section(get_features, display_features),
-    Section(get_supervised, display_supervised),
-    Section(get_citation, display_citation),
-]
-
-%>
-
-## --------------------------- Single builder ---------------------------
-
-<%def name="display_builder(builder, sections)">\
-% for section in sections:
-${section.make(builder)}\
-% endfor
-</%def>
-
-## --------------------------- Builder configs ---------------------------
-
-<%def name="display_all_builders(builders)">\
-<%
-
-# For each fields, extract if the field is shared or unique accross builder.
-common_sections = []
-unique_sections = []
-for section in all_sections:
-  if len(set(section.get_signature(b) for b in builders)) == 1:
-    common_sections.append(section)
-  else:
-    unique_sections.append(section)
-
-%>
-
-${display_builder(next(iter(builders)), common_sections)}
-
-% for i, builder in enumerate(builders):
-<%
-header_suffix = '(default config)' if i == 0 else ''
-%>\
-${'##'} ${builder.name}/${builder.builder_config.name} ${header_suffix}
-
-${display_builder(builder, unique_sections)}
-% endfor
-</%def>
-
-## --------------------------- Main page ---------------------------
-
-${'#'} `${builder.name}`
-
-%if builder.MANUAL_DOWNLOAD_INSTRUCTIONS:
-Warning: Manual download required. See instructions below.
-%endif
-
-<%doc>First case: Single builder.</%doc>\
-% if not builder.builder_config:
-${display_builder(builder, all_sections)}
-<%doc>Second case: Builder configs.</%doc>\
-% else:
-${display_all_builders(config_builders)}
-% endif
+<%!

```

Comment:
Could you fix the diffs ? I can't see what has changed here.

Reply:
I think because of adding figure it shows this difference as I didn't change anything here

Reply:
Could you search on the internet if there are solutions and why those diffs are showing if those lines hasn't been changed. Does `git diff` works ?

Reply:
Sure

Reply:
This happens because I  replace the same file from one place to other although no changes in inside of file to solve this I have to revert back whole commits

Reply:
I think you can also mark the file as branched afterward

Reply:
Should I also increase quality of images as I keep it to 30 now ?